packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -24,52 +27,379 @@ const animateIndicator = (indicator, animate) => {
+                        color: var(--spectrum-global-color-gray-900);
+                        box-shadow: 0 3px 16px #46303029;
+                        border-radius: var(--spectrum-global-dimension-size-50);
+                    }
+                    img {
+                      max-width: var(--spectrum-global-dimension-size-500);
+                      max-height: var(--spectrum-global-dimension-size-500);
+                    }
+                  `}
+                />
+              )
+            })}
+            <TabsIndicator ref={selectedTabIndicator} />
+          </Tabs>
+        : null}
+        {menuItems?.length > 0 ?
+          menuItems.map((data, index) => {
+            const isHidden = selectedIndex.tab === index;
+            return(
+              <TabView
+                key={`tabView_${index}`}
+                className={"tabView"}
+                isHidden={!isHidden}
+                css={css`
+                  text-align: left;
+                  width:${layoutColumns(8)};
+                  padding-left:var(--spectrum-global-dimension-size-700);
+                  padding-right:var(--spectrum-global-dimension-size-200);
+
+                  @media only screen and (max-width: 480px) {
+                    padding-left: inherit !important;
+                    max-width: ${layoutColumns(3)} !important;
+                  }
+
+                  @media only screen and (device-width: ${MOBILE_SCREEN_WIDTH}) {
+                    max-width: ${layoutColumns(3.5)} !important;
+                  }
+                  @media only screen and (device-width: ${TABLET_SCREEN_WIDTH}) {
+                    max-width: ${layoutColumns(6.5)} !important;
+                    padding-left:var(--spectrum-global-dimension-size-500);
+                  }
+
+                `}
+
+              >
+                {data['tabViewContent'] ? data['tabViewContent'] : null}
+              </TabView>
+            )
+          })
+        : null}
+      </div>
+    </section>
+  )
+}
+
+
+Tabs.propTypes = {
+  orientation: PropTypes.oneOf(["horizontal", "vertical"]),
+  density: PropTypes.oneOf(["regular", "compact"]),
+  isQuiet: PropTypes.boolean,
+  className: PropTypes.string,
+};
+
+Item.propTypes = {
+  isSelected: PropTypes.boolean,
+  isDisabled: PropTypes.boolean,
+  className: PropTypes.string,
+  icon: PropTypes.element,
+  label: PropTypes.object
+};
+
+TabItemIcon.propTypes = {
+  elementType: PropTypes.string,
+  icon: PropTypes.element,
+  isSelected: PropTypes.boolean,
+  isDisabled: PropTypes.boolean,
+  iconSize: PropTypes.string,
+  className: PropTypes.string
+};
+
+TabView.propTypes = {
+  isHidden: PropTypes.boolean,
+  className: PropTypes.string
+};
+
+TabsBlock.propTypes = {
+  theme: PropTypes.oneOf(["light", "dark", "lightest"]),
+  orientation: PropTypes.oneOf(["horizontal", "vertical"]),
+  heading: PropTypes.string,
+  tabViewContent: PropTypes.string,
```

Comment:
Can you rename `tabViewContent` to just `content` to make the slot name more generic ? I can imagine other components that can make use of such a `content` slot.

Reply:
This is done.